Week-6/Assignment_6.1_MaxSum.js

```diff
@@ -0,0 +1,36 @@
+const maxSubArray = (nums) => {
+    // initiate two variable, maxSum for total max, sum for current max
+    let maxSum = -Infinity
+    let currentSum = 0
+    // iterate through the nums, store sub-problems result
+    for(let i = 0; i < nums.length; i++){ 
+        //cumulating answers to the top
+        const maxSubArray = (nums) => {
+    // initiate two variable, maxSum for total max, sum for current max
+    let maxSum = -Infinity

```

Comment:
Indentation is wrong, use any VSCode extension, build a habit of having proper indentation